src/screens/Calendar.tsx

```diff
@@ -1,22 +1,45 @@
-import React from 'react'
-import { StatusBar, StyleSheet, Text, View } from 'react-native'
-import { Agenda } from 'react-native-calendars'
+import React, {useState} from 'react'
+import { StatusBar, StyleSheet, Text, View, ScrollView } from 'react-native'
+import { Agenda, CalendarProvider } from 'react-native-calendars'
 
 export default function Calendar(): JSX.Element {
+    var date = new Date();
+    const initialDate = date.toISOString().substr(0,10)
+	var currentDateSelected = initialDate
     return (
         <View style={{marginTop: StatusBar.currentHeight}}>
-            <View style={{height: '100%'}}>                
-                <Agenda renderList={() => DailyData()}/>
+            <View style={{height: '100%'}}>
+                <CalendarProvider date={initialDate}>             
+                <Agenda renderList={() => DailyData(currentDateSelected)}
+                    onDayPress={day => {   
+                        currentDateSelected = day.dateString  
+                    }}
+                    hideExtraDays={true}
+                    theme={{
+                        agendaKnobColor: '#5838B4',
+                        calendarBackground: 'black',
+                        selectedDayBackgroundColor: "#5838B4",
+                        monthTextColor: 'white',
+                        todayTextColor: '#5838B4',
+                        dayTextColor: 'white'
+                    }}
+                />
+                </CalendarProvider> 
             </View>
         </View>
     )
 }
 
-function DailyData(): JSX.Element {
+function DailyData(day): JSX.Element {
+
     return (
-        <View style={{alignItems: 'center', padding: 10}}>
-            <Text style={{fontSize: 20}}>Daily Data</Text>
+        <View style={{padding: 10}}>
+            <Text style={{textAlign: 'center', fontSize: 20}}>Daily Data</Text>
             <View style={styles.lineSeperator}/>
+                <ScrollView>
+                    {//Display data for Date
+                    }
```

Comment:
I know this is temporary, but the new line feels unnecessary
```suggestion
                    {/* TODO: Display data for Date */}
```